src/mixer/basetrackplayer.cpp

```diff
@@ -38,31 +38,35 @@ BaseTrackPlayer::BaseTrackPlayer(QObject* pParent, const QString& group)
         : BasePlayer(pParent, group) {
 }
 
-BaseTrackPlayerImpl::BaseTrackPlayerImpl(QObject* pParent,
+BaseTrackPlayerImpl::BaseTrackPlayerImpl(
+        QObject* pParent,
         UserSettingsPointer pConfig,
         EngineMaster* pMixingEngine,
         EffectsManager* pEffectsManager,
         VisualsManager* pVisualsManager,
         EngineChannel::ChannelOrientation defaultOrientation,
-        const QString& group,
+        const ChannelHandleAndGroup& handle_group,
         bool defaultMaster,
         bool defaultHeadphones,
         bool primaryDeck)
-        : BaseTrackPlayer(pParent, group),
+        : BaseTrackPlayer(pParent, handle_group.name()),
```

Comment:
Could you make BaseTrackPlayer also store the ChannelHandleAndGroup?

Reply:
I could, but this would require many more changes, and I don't want to go too deeply into this for now.